### ISSUE TITLE
switch from keras to tf.keras on tensorflow 2

### ISSUE THREAD

**Issue by timodonnell** [openvax/mhcflurry#151]
Sounds like that is the Keras recommendation: https://github.com/keras-team/keras/releases/tag/2.3.0

**Comment by bionicles** [comment]
Hi, I’m hitting this issue, pip install mhcflurry fails today on ubuntu 18.04 python 3.8

**Comment by bionicles** [comment]
the issue occurs if the latest version of python (3.8) is installed. mhcflurry needs tf 1.15 which doesn't support python 3.8. downgrade to py37 allowed me to install mhcflurry (but I had to downgrade tensorflow)

@timodonnell which files use keras? we could swap it out pretty quick i bet

**Comment by timodonnell** [comment]
Thanks, @bionicles . Good to know about py 3.8. We should note this in the docs.

Unfortunately the tf 2 API has quite a few changes, and I'm not sure if the model serialization is backward compatible. It's also pretty easy to inadvertently introduce changes that affect model accuracy in subtle ways. If you have time to take a shot though, we'd definitely consider a PR if you're able to do it.

**Comment by bionicles** [comment]
For now we’re full steam ahead on CRISPR engineering but I’d imagine this is doable and wouldn’t screw up too much stuff
